plugins.json

```diff
@@ -4,6 +4,14 @@
       "description": "A Pulumi plugin that provides Pulumi pipeline stage as a pre-configured job.",
       "provider": "https://github.com/pulumi",
       "releases": [
+        {
+          "version": "v0.1.0",
+          "date": "2021-01-29T00:42:50.629739Z",
+          "requires": "orca>=0.0.0,deck>=0.0.0",
+          "sha512sum": "9fbd3f20985cbd03f52468af92588037662a62dab8c54437f16c86be5a4e7717aa585094db0e47ef0941008f287220109159c0ec20a38f8ff1e6a7a7a08d7ae8",
```

Comment:
The SHA512 sum is output into a metadata JSON file when you run `./gradlew releaseBundle` for a particular version. I ran this locally and found this value. The same with the `requires` attribute too. As for the URL, it comes from the [input](https://github.com/pulumi/spinnaker-preconfigured-job-plugin/runs/1787455460?check_suite_focus=true#step:10:4) to the GH action that failed. Also it is just the URL for the asset uploaded to the GH [release](https://github.com/pulumi/spinnaker-preconfigured-job-plugin/releases/tag/v0.1.0).